<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addBookClassClass</class>
 <widget class="QMainWindow" name="addBookClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加图书</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/7b78ceea29ae3cd6dbf27aa34bac4892.jpeg</normaloff>icon/7b78ceea29ae3cd6dbf27aa34bac4892.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="bookNameEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>书名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="articleEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>作者</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>类型</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="localEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numbEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>数量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
